OpenGL changes:

udhos opened the pull request:
- Make setRotationMatrix() compatible, in respect to inversion, with setViewMatrix().
- Add helper setModelMatrix().
- Add slightly simpler/faster setViewMatrix2().

johnmccutchan commented on lib/src/vector_math/opengl.dart:
```diff
@@ -59,12 +59,64 @@ part of vector_math;
-void setRotationMatrix(
-    Matrix4 rotationMatrix, Vector3 forwardDirection, upDirection) {
+void setRotationMatrix(Matrix4 rotationMatrix, Vector3 forwardDirection,
+                       Vector3 upDirection) {
```
fix formatting

johnmccutchan commented on lib/src/vector_math/opengl.dart:
```diff
@@ -59,12 +59,64 @@ part of vector_math;
+ * View transformation is the inverse of the model transformation.
+ * View matrix is commonly used to compute the camera location/orientation into the full model-view stack.
+ * 
+ * Sligthly simpler/faster implementation for setViewMatrix() below.
```
Should this just replace setViewMatrix?

udhos commented:
John,
1) fixed the formatting, but the formatter affected other lines as well.
2) replaced setViewMatrix() with the new version.
Thanks,
Everton

udhos commented:
Please take another look at this request.
